src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs

```diff
@@ -206,13 +225,33 @@ private void Stop()
 
                 IPEndPoint localEndPoint = MsQuicAddressHelpers.INetToIPEndPoint(ref *(SOCKADDR_INET*)connectionInfo.LocalAddress);
                 IPEndPoint remoteEndPoint = MsQuicAddressHelpers.INetToIPEndPoint(ref *(SOCKADDR_INET*)connectionInfo.RemoteAddress);
+                string targetHost = string.Empty;   // compat with SslStream
+                if (connectionInfo.ServerNameLength > 0 && connectionInfo.ServerName != IntPtr.Zero)
+                {
+                    // TBD We should figure out what to do with international names.
```

Comment:
We should file an issue to track and list it here.

(What do we do for SslStream? Maybe SCHANNEL handles this for us, but does OpenSsl?)

Reply:
I will need to investigate. We have helper in SslStream but I was wondering if MsQuic already does any other there Some of the comments looked like it may expect utf8. I will file issue to follow-up on this.

Reply:
created https://github.com/dotnet/runtime/issues/55687 for tracking.

Reply:
Should we than get rid of the TBD, or at least mention the issue number there?